### PR TITLE
Base implementation of ratings service

### PR DESCRIPTION
**Pull request by cooperative0zero**
Three layers are implemented: database access layer, business logic layer (services) and controllers layer. 
Spring Data JPA was used to organize work with the database, Liquibase was used to manage migrations.
Endpoints for trip evaluation have been created.

### CODE REVIEW

File: src/main/java/com/modsen/software/ratings/controller/RatingController.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+@RequestMapping("/api/v1/ratings")
+public class RatingController {
+
+    private final RatingService ratingService;
+
+    @GetMapping
+    public ResponseEntity<List<RatingResponse>> getAll(@RequestParam(required = false, defaultValue = "0") @Min(0) Integer pageNumber,
+                                                       @RequestParam(required = false, defaultValue = "10") @Min(1) Integer pageSize,
+                                                       @RequestParam(required = false, defaultValue = "id") String sortBy,
+                                                       @RequestParam(required = false, defaultValue = "desc") String sortOrder) {
```
Comment (psidarovich) [inline src/main/java/com/modsen/software/ratings/controller/RatingController.java]: Minor: use next line, please

File: src/main/java/com/modsen/software/ratings/controller/RatingController.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        List<RatingResponse> response = ratingService.getAll(pageNumber, pageSize, sortBy, sortOrder);
+        return new ResponseEntity<>(response, HttpStatus.OK);
+    }
+
+    @GetMapping("/passengers/{id}")
+    public ResponseEntity<List<RatingResponse>> getAllByPassengerId(@PathVariable @Min(1) Long id,
```
Comment (psidarovich) [inline src/main/java/com/modsen/software/ratings/controller/RatingController.java]: If you use pagination - return not just list of items, but structure:
```
{
 "items": ...,
 "page": 1,
 "size": 20,
 "total": 200
}
```

**Review by psidarovich — APPROVED** [review]

File: src/main/java/com/modsen/software/ratings/controller/RatingController.java (hunk 1 above)
Comment (cooperative0zero) [reply src/main/java/com/modsen/software/ratings/controller/RatingController.java]: fixed

File: src/main/java/com/modsen/software/ratings/controller/RatingController.java (hunk 2 above)
Comment (cooperative0zero) [reply src/main/java/com/modsen/software/ratings/controller/RatingController.java]: fixed

**Pull request merged**